PyDSS/pydss_project.py

```diff
@@ -31,9 +31,10 @@
 
 class PyDssProject:
     """Represents the project options for a PyDSS simulation."""
-    def __init__(self, path, name, scenarios, simulation_config, fs_intf=None):
+    def __init__(self, path, name, scenarios, simulation_config, fs_intf=None, simulation_file=None):
         self._name = name
         self._scenarios = scenarios
+        self._simulation_file = simulation_file
```

Comment:
I think this might simplify some code:
```
if simulation_file is None:
    self._simulation_file = os.path.join(self._project_dir, SIMULATION_SETTINGS_FILENAME)
else:
     self._simulation_file = simulation_file
```

Reply:
fixed